slices/erlang-base.yaml

```diff
@@ -0,0 +1,51 @@
+package: erlang-base
+
+essential:
+  - erlang-base_copyright
+
+slices:
+  bins:
+    essential:
+      - adduser_bins
+      - erlang-base_config
+      - erlang-base_libs
+      - erlang-base_releases
+      - libc6_libs
+      - libgcc-s1_libs
+      - libstdc++6_libs
+      - libsystemd0_libs
+      - libtinfo6_libs
+      - procps_bins
```

Comment:
Unlike adduser, I'm not very confortable removing procps, because although it may not affect rabbitmq-server, it may affect other packages that depends on erlang* and need to run erlang modules that executes one of the procps binaries.

Reply:
But did you findd any use-cases for the procps bins in any of those erlang-* packages?

Reply:
not really. I'll remove it for now.